src/olympia/amo/tests/test_helpers.py

```diff
@@ -156,8 +157,18 @@ def test_url(mock_reverse):
 
 
 def test_url_src():
-    s = render('{{ url("addons.detail", "a3615", src="xxx") }}')
-    assert s.endswith('?src=xxx')
+    fragment = render('{{ url("addons.detail", "a3615", src="xxx") }}')
+    assert fragment.endswith('?src=xxx')
+
+
+def test_drf_url():
+    rf = RequestFactory()
+    request = rf.get('/hello/')
+    fragment = render('{{ drf_url("addon-detail", pk="a3615") }}',
+                      context={'request': request})
+    # without a /vX/ in the request, RESTFRAMEWORK['DEFAULT_VERSION'] is used.
+    assert fragment == jinja_helpers.absolutify(
+        reverse('v4:addon-detail', args=['a3615'], add_prefix=False))
```

Comment:
This doesn't test the helper without the request specified.

Reply:
The `context` is added by django when it does the template render.  (It'll fail to find "addon-detail" without the request because reverse can't evaluate it without the namespacing.  If we _do_ want to test it anyway, not sure what we'd be testing: it fails with an exception?)

Reply:
Yeah, having a test that documents that it properly fails with an expected exception is what I'd like to see.

Reply:
https://github.com/mozilla/addons-server/pull/8229/commits/f59f8efcc27229a68431b09056f00c9e67f8c33a